### ISSUE TITLE
Nigeria: Ensure all members in Wikidata have NASS IDs

### ISSUE THREAD

**Issue by tmtmtmtm** [everypolitician/everypolitician#605]

**Comment by lucychambers** [comment]
Do I assume that the presence of https://www.wikidata.org/wiki/Wikidata:Database_reports/Constraint_violations/P4139 - means that something has happened on this front, or do we need to check that there isn't anyone missing?

**Comment by tmtmtmtm** [comment]
> Do I assume that the presence of https://www.wikidata.org/wiki/Wikidata:Database_reports/Constraint_violations/P4139 - means that something has happened on this front

No, that is the inverse of this issue. That is a standard Wikidata report of the type that gets produced for every Property (e.g. https://www.wikidata.org/wiki/Wikidata:Database_reports/Constraint_violations/P39), and reports on people who _have_ the property set not people who _don't_.
